src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -178,19 +182,22 @@ data:
         idTokens: 30m
       customer:
         enabled: true
-        connectorName: test
-        connectorType: microsoft
-        connectorConfig: >-
-          clientID: <CLIENT-ID-SET-IN-YOUR-IdP>
-          clientSecret: <CLIENT-SECRET-SET-IN--YOUR-IdP>
-          tenant: <TENANT-SET-SET-IN--YOUR-IdP>
-          redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
+        connectors:
+        - id: customer
+          connectorName: test
+          connectorType: microsoft
+          connectorConfig: >-
+            clientID: <CLIENT-ID-SET-IN-YOUR-IdP>
+            clientSecret: <CLIENT-SECRET-SET-IN--YOUR-IdP>
+            tenant: <TENANT-SET-SET-IN--YOUR-IdP>
+            redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
 ```
 
 {{< /tab >}}
 {{< /tabs >}}
 
 __Note__: In the above snippet you have to replace the `<CLUSTERID>` variable and add the Kubernetes Certificate Authority to ensure Dex can trust the API endpoint. Finally you have to use a connector. Here we show examples for Keycloak, Active Directory, and GitHub.
+You can use more than one connector, but they need to have a different `id` value. We advice to use `- id: customer` for your primary connector.
```

Comment:
If this is supposed to be a separate paragraph, it needs another line break before it.

Reply:
I think it's fine in the same paragraph since it refers to the same topic as the previous sentence. :)

Reply:
ok